Voycar.Api.Web/Features/Users/Endpoints/Get/WhoAmI/Endpoint.cs

```diff
@@ -0,0 +1,40 @@
+namespace Voycar.Api.Web.Features.Users.Endpoints.Get.WhoAmI;
+
+using Repository;
+
+public class Endpoint : Endpoint<Request, Results<Ok<Response>, BadRequest<string>>, Mapper>
+{
+    private readonly IUsers _userRepository;
+
+    public Endpoint(IUsers userRepository)
+    {
+        this._userRepository = userRepository;
+    }
+
+
+    public override void Configure()
+    {
+        this.Get("user/whoami");
+        this.Roles("member", "employee", "admin");
+        this.Summary(summary =>
+        {
+            summary.Summary = "Get Email of logged in user";
+            summary.Description = "Get the Email of the user who is logged in according to the request cookie";
+            summary.Responses[200] = "If user is found";
+            summary.Responses[400] = "If somehow the user is logged in but can't be found";
```

Comment:
```suggestion
            summary.Responses[400] = "If the user somehow can't be found";
```
Ich denke, es sollte schon klar sein, dass der User eingeloggt ist, sonst würde er ja einen anderen Fehler bekommen.